scipy/stats/stats.py

```diff
@@ -2492,7 +2492,7 @@ def sigmaclip(a, low=4., high=4.):
     return SigmaclipResult(c, critlower, critupper)
 
 
-def trimboth(a, proportiontocut, axis=0):
+def trimboth(a, proportiontocut=0.2, axis=0):
```

Comment:
We're anyway making other backwards incompatible changes there (at least I'd like to), so let's just add this one to that. Shouldn't be a big deal.